maintaining.md

```diff
@@ -0,0 +1,77 @@
+# Maintainers Guide
+
+Note that this document is intended for the owners of this repository who are
+maintaining the common images (in the `arch` family), and the gce Pacman repo.
+If you're just looking to build your own image, follow the instructions at the
+end of README.md.
+
+## Deploying Public Images
+
+```console
+$ DATE=$(date --utc +%Y%m%d)
+
+$ sudo pacman -S --needed arch-install-scripts e2fsprogs
+
+$ sudo ./build-arch-gce
+
+$ gcloud auth login
+
+$ gcloud config set project arch-linux-gce
+
+$ gsutil cp "arch-v${DATE}.tar.gz" gs://arch-linux-gce
+
+$ gcloud compute images create "arch-v${DATE}" \
+    --source-uri="gs://arch-linux-gce/arch-v${DATE}.tar.gz" \
+    --guest-os-features=VIRTIO_SCSI_MULTIQUEUE \
+    --family=arch
```

Comment:
I also add (the somewhat redundant) "--description="Arch-Linux built on ${DATE}."

Reply:
Done